infrastructure/org-policies/project-services.tf

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_project_service" "project" {
+  project = var.project
+  for_each = toset(var.gcp_project_services)
+  service  = each.value
+
+  timeouts {
+    create = "30m"
+    update = "40m"
+  }
+  
+  # Ensure service is truly active before continuing onward
+  provisioner "local-exec" {
```

Comment:
oh I like this! Run into this problem all the time

Reply:
From everything I've read, this `local-exec` shouldn't be necessary? 🤔

Reply:
@markmandel   

Unfortunately, this hack is still necessary.  The orgpolicy API is flakey and in like 1 out of 4 runs is not actually ready when Terraform marks it as ready, which in turn causes the org policies to fail to be created

Reply:
That;s  a shame, but fair enough.